### PR TITLE
Dev experience 2022

### PR DESCRIPTION
**Pull request by r41k0u**

### CODE REVIEW

File: _posts/2022/2022-01-26-dev-experience-2022.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+## The Second Semester
+
+#### The initiation
+
+I got a phone call informing me about my induction to SDSLabs. We had an intro chapo (online, coz covid) and got to know everyone there. We were added on the workspace apps and this got the cart rolling.
+
+The semester was filled with lectures on various subjects (Git, Bash scripting, Web development, Game development) and we has follow up assignments. The follow up assignments spurred me to learn about something new and implement it within a deadline.
```
Comment (Scar26) [inline _posts/2022/2022-01-26-dev-experience-2022.md:42]: The first semester
~and we has follow up assignments~ and corresponding assignments.

File: _posts/2022/2022-01-26-dev-experience-2022.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+After this, we were explained about all the apps and projects going on in SDSLabs. We were briefed about their status, what technology they are built upon, what problems they solve and what more work needs to be done.
```
Comment (Scar26) [inline _posts/2022/2022-01-26-dev-experience-2022.md:44]: After the assignment season was concluded, we were introduced to the active apps and projects in SDSLabs. We were briefed on the status for each project and a received an overview of the tech stack, ongoing work, future issues and overall vision.

File: _posts/2022/2022-01-26-dev-experience-2022.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+By now we started participating in various hackathons (*and managing to win some of them!!*). Members would come together, form teams among themselves (sometimes even our alumni joined us) and we would participate in the hackathons. Sitting through hours of code sprints and preparing POCs, those were exhilarating hours.
+
+#### Majors
+After the minors, we had to choose our Majors: solving a large issue or implementing something important in the ongoing projects. We again met for the sorting and were assigned the responsibilities.
+
+#### Releasing a Project
+
+When a project comes to fruition and is ready to be open-sourced, everyone at SDSLabs is pretty excited about it. Seeing the hardwork of our members getting released ittruly a moment to be cherished. I witnessed the release of [ZapDB](https://github.com/sdslabs/zap-db) and [Studyportal](https://github.com/sdslabs/studyportal).
+
+#### Information Security in SDSLabs
```
Comment (Scar26) [inline _posts/2022/2022-01-26-dev-experience-2022.md:63]: A separate section for security seems a little unnecessary. If the point is to illustrate labs branching out into non-dev fields, highlighting one of said fields presents said diversification through a very restricted lens by undermining other areas of tech that our members have explored/are exploring. BackdoorCTF toh uda de imo but if you wanna write about your experience playing CSAW in a team with other labs folk or highlight your achievements, I would suggest adjusting that elsewhere and then merge this into the next section, expanding upon it by mentioning people around you working in areas which aren't strictly de. You can bring up ML, research, hardware, CS theory e.t.c in addition to security so it feels more inclusive (interest wise) (You cold also use your CTF experience as an example of exploration in this section as opposed to being the highlight of it)

**Review by Scar26 — CHANGES_REQUESTED** [review]

File: _posts/2022/2022-01-26-dev-experience-2022.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+A few days later, I got a call for interview. The interview was conducted in 2 parts. First one was basic problem solving, and we also discussed basic networking and webapp vulnerabilities.
+
+The second round was based on my interests and my ability to tackle real world problems. I was asked to come up for high levels designs for Tic-Tac-Toe and a library management system. This was also a fanboy moment for me as one of the interviewers was Twarit Waikar (SDSLabs 2021 alumnus), a game developer who had worked extensively on SDSLabs's game engines Rubeus and Rootex. I had read his blogs and looked at his games and the game engines and was thoroughly impressed by his work (Making a game engine seemed like a very huge undertaking to me). At the end of this round, I was given time to let loose with any queries I had for SDSLabs, an opportunity I made good use of. I asked them about game engines, game development and web security.
```
Comment (Scar26) [inline _posts/2022/2022-01-26-dev-experience-2022.md:34]: ~come up for~ come up with

File: _posts/2022/2022-01-26-dev-experience-2022.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+#### Majors
+After the minors, we had to choose our Majors: solving a large issue or implementing something important in the ongoing projects. We again met for the sorting and were assigned the responsibilities.
+
+#### Releasing a Project
+
+When a project comes to fruition and is ready to be open-sourced, everyone at SDSLabs is pretty excited about it. Seeing the hardwork of our members getting released ittruly a moment to be cherished. I witnessed the release of [ZapDB](https://github.com/sdslabs/zap-db) and [Studyportal](https://github.com/sdslabs/studyportal).
+
+#### Not just Dev
+
+It's not always software develpoment that we work on. We also work on IoT projects to make our workspace smarter (like presence detection using computer vision, WiFi locks). We have members passionate about information security and Data Science. We also have paper discussions in which we all read a research paper on related to CS and discuss it with other members.
```
Comment (Scar26) [inline _posts/2022/2022-01-26-dev-experience-2022.md:65]: iske end me add kar de apna CSAW experience as an example

**Review by Scar26 — CHANGES_REQUESTED** [review]

File: _posts/2022/2022-01-26-dev-experience-2022.md (hunk 5 above)
Comment (sin3point14) [reply _posts/2022/2022-01-26-dev-experience-2022.md:65]: Ig you can even write fun things like playing Minecraft together? :p

Comment (r41k0u) [reply _posts/2022/2022-01-26-dev-experience-2022.md:65]: minesweeper aur minecraft aur nioh ka soch rha tha lekin lamba ho jaata

**Pull request merged**